docs/millennium-os/manual/chapters/40_gcodes.md

```diff
@@ -36,7 +36,7 @@ When using multiple milling tools, we must compensate for length differences bet
 G37.1
 ```
 
-When there is no toolsetter available, it is necessary to re-zero the Z origin after changing tools - because the new tool will never be installed with exactly the same length as the previous one.
+When there is no toolsetter available, it is necessary to re-zero the Z origin after changing tools - because the new tool will never be installed with the same length as the previous one.
```

Comment:
```suggestion
When there is no toolsetter available, it is necessary to re-zero the Z-Axis origin after changing tools - because the new tool will never be installed with the same length as the previous one.
```